### ISSUE TITLE
Problems using "multiSelectionKeyCode" in combination with "useKeyPress"

### ISSUE THREAD

**Issue by zyh374892663** [wbkd/react-flow#2203]
### Describe the Bug

According to the user's usage habits, ctrl + left button is usually used for multi-selection. If multiSelectionKeyCode is not set, Meta + left button is OK. After setting, it is invalid.

### Steps to Reproduce the Bug or Issue

```
import React from "react";
import ReactFlow, { addEdge, MiniMap, Controls, Background, useNodesState, useEdgesState, useKeyPress } from "react-flow-renderer";

import { nodes as initialNodes, edges as initialEdges } from "./initial-elements";

const onInit = (reactFlowInstance) => console.log("flow loaded:", reactFlowInstance);

const FlowchartV2 = () => {
    const [nodes, setNodes, onNodesChange] = useNodesState(initialNodes);
    const [edges, setEdges, onEdgesChange] = useEdgesState(initialEdges);
    const onConnect = (params) => setEdges((eds) => addEdge(params, eds));
    // const controlPressed = useKeyPress("Control");
    // const strgPressed = useKeyPress("Strg");
    // const controlLeftPressed = useKeyPress("ControlLeft");
    const ctrlPressed = useKeyPress("Ctrl");

    return (
        <ReactFlow
            nodes={nodes}
            edges={edges}
            onNodesChange={onNodesChange}
            onEdgesChange={onEdgesChange}
            onConnect={onConnect}
            onInit={onInit}
            fitView
            attributionPosition="top-right"
            // multiSelectionKeyCode={controlPressed}
            // multiSelectionKeyCode={strgPressed}
            // multiSelectionKeyCode={controlLeftPressed}
            multiSelectionKeyCode={ctrlPressed}
            // multiSelectionKeyCode="Ctrl"
            // multiSelectionKeyCode="Control"
            // multiSelectionKeyCode="ControlLeft"
            // multiSelectionKeyCode="Strg"
        >
            <MiniMap
                nodeStrokeColor={(n) => {
                    if (n.style?.background) return n.style.background;
                    if (n.type === "input") return "#0041d0";
                    if (n.type === "output") return "#ff0072";
                    if (n.type === "default") return "#1a192b";

                    return "#eee";
                }}
                nodeColor={(n) => {
                    if (n.style?.background) return n.style.background;

                    return "#fff";
                }}
                nodeBorderRadius={2}
            />
            <Controls />
            <Background color="#aaa" gap={16} />
        </ReactFlow>
    );
};

export default FlowchartV2;

```

### Expected behavior

Expect to use ctrl + left click for multiple selection

### Platform

- OS: [e.g. macOS, Windows, Linux]
- Browser: [e.g. Chrome, Safari, Firefox]
- Version: [e.g. 91.1]
Windows
Chrome
^10.3.4

**Comment by moklick** [comment]
In your code example you are passing a boolean to the `multiSelectionKeyCode` prop which is not valid. I can see that you also tried to do `multiSelectionKeyCode="ControlLeft"` which should work but triggers a right click. I will see what we can do about this.

**Comment by zyh374892663** [comment]
The `multiSelectionKeyCode` parameter takes effect only when `Meta` is set, and other keyboards are invalid. The browser default event and canvas ContextMenuEvent hinder the right click. Expect to use the keyboard + left mouse button to perform multiple selection operations on `node` and `edge`

**Comment by zyh374892663** [comment]
@moklick Excuse me, is this a code problem? What is the current progress?

**Comment by moklick** [comment]
@zyh374892663 I haven't had time to look into it yet.

**Comment by rolox05** [comment]
I'm still facing the same issue with multiSelectionKeyCode after we've migrated to V10. It only works on Mac (or 'Meta' key). 

is there any progress on this already?

**Comment by moklick** [comment]
This is fixed in v10.3.7.

Something like this should work now:

`multiSelectionKeyCode={isWindows ? 'ControlLeft' : 'Meta'}`

**Comment by rolox05** [comment]
Good morning @moklick, multi selection for nodes is fixed up to version V10.3.7 on windows, but we're still missing multi selection of edges. 

Do you want me to fill a different issue for that scpecifically or this  one is ok?

**Comment by moklick** [comment]
@rolox05 please open another issue for that.
